pegasus/src/database.rb

```diff
@@ -1,5 +1,8 @@
 require 'cdo/db'
+require 'cdo/geocoder'
 require 'cdo/properties'
+require 'json'
+require 'securerandom'
```

Comment:
Note that it isn't clear to me where this is being used. At the same time, I'm scared to remove it, as it could break some file requiring it via transitivity.